kotlin/.gitignore

```diff
@@ -0,0 +1,10 @@
+## Gradle
```

Comment:
These are Gradle specific ignores for Git. Shall we keep it here or add it to the top level `.gitignore`?

Reply:
keep it here